cmd/clusterctl/cmd/delete.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Kubernetes Authors.
+Copyright 2019 The Kubernetes Authors.
```

Comment:
This probably just happened as part of a file move, but I don't think we're supposed to change license header dates for existing files?

Reply:
I'm rebuilding from scratch, so I think 2019 is fine despite the fact this very specific file existed in previous versions